bin/changelog/github/requests.js

```diff
@@ -0,0 +1,90 @@
+'use strict';
+
+const { REPO } = require( '../config' );
+const { authedGraphql } = require( '../common' );
+
+/* eslint no-console: 0 */
+
+const getMilestoneNumber = async ( version ) => {
+	const [ owner, repo ] = REPO.split( '/' );
+	const query = `
+	{
+		repository(owner: "${ owner }", name: "${ repo }") {
+			milestones(last: 50) {
+				nodes {
+					title
+					number
+				}
+			}
+		}
+	}
+	`;
+	const data = await authedGraphql( query );
+	const matchingNode = data.repository.milestones.nodes.find(
+		( node ) => node.title === version
+	);
+	if ( ! matchingNode ) {
+		throw new Error(
+			`Unable to find a milestone matching the given version ${ version }`
+		);
+	}
+	return matchingNode.number;
+};
+
+const getQuery = ( milestoneNumber, before ) => {
+	const [ owner, repo ] = REPO.split( '/' );
+	const paging = before ? `, before: "${ before }"` : '';
+	return `
+	{
+		repository(owner: "${ owner }", name: "${ repo }") {
+			milestone(number: ${ milestoneNumber }) {
+				pullRequests(last: 100, states: [MERGED]${ paging }) {
+					totalCount
+					pageInfo {
+						hasNextPage
+						endCursor
+					}
+					nodes {
+						number
+						title
+						url
+						author {
+							login
+						}
+						body
+						labels(last: 10) {
+							nodes {
+								name
+							}
+						}
+					}
+				}
+			}
+		}
+	}
+	`;
+};
+
+const fetchAllPullRequests = async ( version ) =>
+	await ( async () => {
+		const milestoneNumber = await getMilestoneNumber( version );
+		const fetchResults = async ( before ) => {
```

Comment:
it seems the script is not working with milestones that have more than 100 PR, try Gutenberg 7.2 (180 PR) and Gutenberg 6.6 (57 PR0) the latter works while the former is stuck on an infinite loop